examples/13_write_dynamic_configuration.cpp

```diff
@@ -10,13 +10,15 @@ using namespace openPMD;
 
 int main()
 {
-    if (!getVariants()["adios2"])
+    if (!getVariants()["hdf5"])
     {
         // Example configuration below selects the ADIOS2 backend
         return 0;
     }
 
     using position_t = double;
+
+#if !__NVCOMPILER // see https://github.com/ToruNiina/toml11/issues/205
```

Comment:
Do you want to filter by NVHPC version, since the issue is fixed in newer releases?
https://github.com/ToruNiina/toml11/issues/205#issuecomment-1487381207

Reply:
I'll replace it with the macro used in `src/config.cpp`:
```cpp
#if !defined(__NVCOMPILER_MAJOR__) || __NVCOMPILER_MAJOR__ >= 23
```